source/GameAction.cpp

```diff
@@ -543,6 +563,9 @@ GameAction GameAction::Instantiate(map<string, string> &subs, int jumps, int pay
 	result.fail = fail;
 	result.failCaller = failCaller;
 
+	for(const auto &it : messages)
+		result.messages.push_back(ExclusiveItem<Message>{{it->Text(subs), it->GetCategory()}});
```

Comment:
What if we wanted to have a game action pick messages from a phrase? Right now this is collapsing the phrase into a single random value, and only that one value will be chosen every time the action is ran (assuming it's a repeatable action). I would imagine it'd be better to pick from the phrase at the time of the action being called.

Reply:
The substitutions aren't being applied to the phrase anyway.